pkg/actors/reminders/reminders.go

```diff
@@ -149,7 +156,12 @@ func (r *reminders) CreateReminder(ctx context.Context, reminder *internal.Remin
 	if err != nil {
 		return fmt.Errorf("error storing reminder: %w", err)
 	}
-	return r.startReminder(reminder, stop)
+	err = r.startReminder(reminder, stop)
+	if err != nil {
+		return err
+	}
+	w3cString := diag.SpanContextToW3CString(span.SpanContext())
+	return r.updateReminderTrack(ctx, reminder.Key(), reminder.RepeatsLeft(), time.Time{}, nil, w3cString)
```

Comment:
is there a scenario possible where the reminder is fired immediately and the track is not updated with the trace state info?

Reply:
Changed to update reminder track before starting reminders.